libass/ass_font.c

```diff
@@ -474,7 +474,7 @@ int ass_font_get_index(ASS_FontSelector *fontsel, ASS_Font *font,
         int face_idx;
         ass_msg(font->library, MSGL_INFO,
                 "Glyph 0x%X not found, selecting one more "
-                "font for (%s, %d, %d)", symbol, font->desc.family,
+                "font for (%*s, %d, %d)", symbol, (int)font->desc.family.len, font->desc.family.str,
```

Comment:
The old version passed in the struct itself instead of the string pointer, so it definitely wasn't fine. In theory I probably could've just added `.str`, but I think it's better practice to explicitly pass the length whenever possible.